docs/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Strapi Cloud Deployment - Strapi Developer Docs
+description: Learn how to deploy your Strapi application on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.html
+---
+
+# Strapi Cloud
+
+::: warning
+Strapi Cloud is currently in Closed Beta release. Features and functionality may change prior to General Availability.
+
+Sign up for the waitlist [here](https://strapi.io/cloud).
+:::
+
+This is a step-by-step guide for deploying your Strapi application on Strapi Cloud.
+
+## Prerequisites
+
+Before you can deploy your Strapi application on Strapi Cloud, you need to have the following prerequisites:
+
+* A [GitHub](https://github.com) account
+* GitHub repository for each Strapi application to be deployed
+
+::: tip
+Each Strapi app must be in a separate repository. Your Strapi Cloud account can link to multiple repositories to deploy multiple apps, if needed.
+:::
+
+## Getting started
+
+1. Navigate to the [Strapi Cloud](https://cloud.strapi.io) login page.
+
+![Strapi Cloud login page](../assets/deployment/cloud/login.png)
+
+2. You are prompted to **Log In with GitHub**. Your Strapi Cloud account is created during this initial login.
+
+3. Once logged in, you will be redirected to the Strapi Cloud **Projects** page. From here you can create your first Strapi Cloud project.
+
+![Strapi Cloud Projects page](../assets/deployment/cloud/projects_empty.png)
+
+### Create a project
+
+1. From the **Projects** page, click the **Create Project** button. You are prompted to **Connect with GitHub**.
+
+::: tip
+Connect the GitHub account that owns the repository or repositories you want to deploy. This can be different from the account that owns the Strapi Cloud account.
```

Comment:
Also, we're releasing [this US](https://strapi-inc.atlassian.net/browse/CLOUD-258) on Monday. Therefore, users will be able to connect multiple organizations (accounts) that are connected to their personal GitHub account. Should we also tell this in this tip?

Maybe we can reword the first sentence to something like this:
"Connect GitHub account or organizations that own repositories you want to deploy".